src/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/ApiCompatServiceProvider.cs

```diff
@@ -33,7 +33,7 @@ internal sealed class ApiCompatServiceProvider
 
                 if (excludeAttributesFiles != null)
                 {
-                    compositeSymbolFilter.Add(new AttributeSymbolFilter(excludeAttributesFiles));
+                    compositeSymbolFilter.Add(new DocIdExcludeListSymbolFilter(excludeAttributesFiles));
```

Comment:
I find `DocIdExcludeListSymbolFilter` quite verbose and hard to read. What about `DocIdSymbolFilter`? The "Exclude" part is unnecessary as the "filter" part in the name already indicates that. I'm not sure what "List" refers to here.

Reply:
Updated